jenkins/scripts/integration_test.sh

```diff
@@ -29,7 +29,7 @@ fi
 
 REPO_ORG="${REPO_ORG:-metal3-io}"
 REPO_NAME="${REPO_NAME:-metal3-dev-env}"
-REPO_BRANCH="${REPO_BRANCH:-master}"
+REPO_BRANCH="${REPO_BRANCH}"
 UPDATED_REPO="${UPDATED_REPO:-https://github.com/${REPO_ORG}/${REPO_NAME}.git}"
 UPDATED_BRANCH="${UPDATED_BRANCH:-master}"
```

Comment:
Do we need to keep default `UPDATED_BRANCH` to master?

Reply:
We can, but I think should be done on a separate PR, because it is out of the scope.